src/runner.rs

```diff
@@ -89,13 +89,24 @@ impl FirefoxRunner {
             profile: prof
         })
     }
+
+    fn has_profile_arg(&self) -> bool {
+        self.args.iter().any(|x| {
+            let arg = x.trim().to_lowercase();
+            arg.starts_with("-profile") ||
+            arg.starts_with("--profile") ||
+            arg.starts_with("/profile")
```

Comment:
Yes.

I guess this doesn't handle `-P` and I don't know if the handling of `-ProfileManager` is right (I guess it is?).